src/lib/domain.js

```diff
@@ -0,0 +1,122 @@
+/**
```

Comment:
This is the plan but I'll do it when I move these from this project to the client (mostly because I'm still a beginner with tests and it takes me some time so I'd rather not delay the component for that).
Next time I'll create tests when creating the helpers :roll_eyes: